examples/basic_mysql_connection/index.js

```diff
@@ -8,13 +8,22 @@ var connection = mysql.createConnection({
 	user: config.user,
 	password: config.password,
 	port: config.port,
-    database: config.database
+  database: config.database
 });
 
 connection.connect();
+let query=['INSERT INTO todos(id,done) VALUES '+mysql.escape(18,0),
+           'UPDATE todos SET name="ali",due="2019-08-31" WHERE id='+mysql.escape(5),
```

Comment:
I'm glad you went through some examples and practice here, but this doesn't really address the problem that the homework was asking for. The homework asked you to `Write the necessary SQL statements to complete the functionality of your application`, and I don't think these SQL queries will help the functionality of your todo application (for example, they don't use any user input, and they all use hard-coded Ids).